docs/topics/multiplatform/multiplatform-run-tests.md

```diff
@@ -1,67 +1,375 @@
-[//]: # (title: Run tests with Kotlin Multiplatform)
+[//]: # (title: Test your multiplatform app − tutorial)
 
-By default, Kotlin supports running tests for JVM, JS, Android, Linux, Windows, macOS as well as iOS, watchOS, and tvOS simulators. 
-To run tests for other Kotlin/Native targets, you need to configure them manually in an appropriate environment, emulator, 
-or test framework.
+In this tutorial, you'll learn how to create, configure, and run tests in Kotlin Multiplatform applications.
 
-## Required dependencies
+> This tutorial assumes that you are familiar with:
+> * The layout of a Kotlin Multiplatform project. If this is not the case,
+    complete [this tutorial](multiplatform-mobile-getting-started.md) to get started.
+> * The basics of popular unit testing frameworks, such as [JUnit](https://junit.org/junit5/).
+>
+{type="tip"}
 
-The [`kotlin.test` API](https://kotlinlang.org/api/latest/kotlin.test/) is available for multiplatform tests. When you
-[create a multiplatform project](multiplatform-library.md), the Project Wizard automatically adds test dependencies to
-common and platform-specific source sets. 
+Tests for multiplatform projects can be divided into two categories:
 
-If you didn't use the Project Wizard to create your project, you can [add the dependencies manually](gradle-configure-project.md#set-dependencies-on-test-libraries).
+* Tests for common code. These tests can be run on any platform using any supported framework.
+* Tests for platform-specific code. These are essential to test platform-specific logic. They use a platform-specific
+  framework and can benefit from its additional features, such as a richer API and a wider range of assertions.
 
-## Run tests for one or more targets
+Both categories are supported in multiplatform projects. This tutorial will show you how to create, set up, and run unit
+tests for both common and platform-specific code.
 
-To run tests for all targets, run the `check` task.
+## Create a sample project
 
-To run tests for a particular target suitable for testing, run a test task `<targetName>Test`.
+1. Check your environment for multiplatform
```

Comment:
Well we can make it a separate section, but it would be just one sentence. Copying the whole Set up page seems to much, and you do need all the tools mentioned in that table.

Reply:
I think at this point (since this tutorial is about testing, not creating apps) users might already have everything set up. So I suggest making it less direct

Reply:
I defer to your expertise in this area 😄